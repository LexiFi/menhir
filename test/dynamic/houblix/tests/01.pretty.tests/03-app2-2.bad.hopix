let h2o = h (o o)
let h2obis = h [o, o]