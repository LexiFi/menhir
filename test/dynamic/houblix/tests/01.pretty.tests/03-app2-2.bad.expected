Line 2, characters 15-16: Error (parsing)
  Syntax error.
